src/sage/tensor/modules/free_module_automorphism.py

```diff
@@ -1055,6 +1056,55 @@ def matrix(self, basis1=None, basis2=None):
                 raise NotImplementedError("basis1 != basis2 not implemented yet")
         return self._matrices[(basis1, basis2)]
 
+    def _some_matrix(self):
+        r"""
+        Return the matrix of ``self`` w.r.t. some basis.
+
+        EXAMPLES::
+
+            sage: M = FiniteRankFreeModule(QQ, 2, name='M')
+            sage: e = M.basis('e')
+            sage: a = M.automorphism([[1,1],[0,2]], name='a')
+            sage: a._some_matrix()
+            [1 1]
+            [0 2]
+        """
+        self.matrix()  # forces the update of the matrix in the module's default
+                       # basis, to make sure that the dictionary self._matrices
+                       # is not empty
+        return next(iter(self._matrices.values()))
+
+    @lazy_attribute
+    def characteristic_polynomial(self):
+        r"""
+        Return the characteristic polynomial of ``self``.
+
+        :meth:`characteristic_polynomial` and :meth:`charpoly` are the same method.
+
+        INPUT:
+
+        - ``var`` -- variable
```

Comment:
```suggestion
        - ``var`` -- string (default: ``'x'``); a variable name
```
as in `minimal_polynomial`